portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -91,7 +91,9 @@ private void putBlogsInDatastore(String tag, String message, String nickname, Li
       String comment = (String) entity.getProperty("text");
       String nickname = (String) entity.getProperty("nickname");
       String email = (String) userService.getCurrentUser().getEmail();
-      long parentID = (long) entity.getProperty("parentID");
+      long parentID = (long) 0;
+     // TODO: UPDATE WHEN FRONTEND INPUT IS TAKEN 
+     // long parentID = (long) entity.getProperty("parentID");
```

Comment:
I think we would usually keep the TODO in line 95, but remove line 96. And then add it for real when we are ready.